manim/mobject/svg/brace.py

```diff
@@ -138,10 +145,12 @@ def __init__(
     ):
         self.label_constructor = label_constructor
         self.label_scale = label_scale
-        VMobject.__init__(self, **kwargs)
+        super().__init__(**kwargs)
+
+        self.basecls = OpenGLVMobject if config["renderer"] == "opengl" else VMobject
```

Comment:
I'd prefer not adding `basecls` as an attribute. Does anything speak against replacing `VMobject` by the corresponding group, i.e., `self.get_group_class()`?

Reply:
Nope! I just didn't think of using it.